src/mesh/mesh.cpp

```diff
@@ -960,6 +960,21 @@ void Mesh::EnrollBndryFncts_(ApplicationInput *app_in) {
     default: // periodic/block BCs handled elsewhere.
       break;
     }
+
+    switch (mesh_bcs[f]) {
```

Comment:
I think that make sense.
Either all variables/swarms use some provided default option, or, user type boundaries conditions have to be implemented for all types.
This would prevent silent bugs where boundaries conditions are not aligned.